cli/io/testConfigHeaders.json

```diff
@@ -0,0 +1,14 @@
+{
+  "credentials": {
+    "http://some.site:8081": "Og=="
+  },
+  "credentialsRequired": false,
+  "headers": {
```

Comment:
this won't support multiple Brooklyn URLs - the `credentials` field is a map of site URL to credentials, if you have multiple sites you will have multiple possible combinations of headers, e.g. one site may use regular Basic auth and one Oauth. Whatever we come up with must be able to cope with that.